CMakeLists.txt

```diff
@@ -5,6 +5,9 @@ project(RealmCore)
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/tools/cmake")
 
+# Load work-around for using ccache with XCode generator
+include(XCodeGeneratorCCacheSupport)
+
```

Comment:
Agreed. This must at least be opt-in rather than opt-out.

Reply:
Agree, this also feels a bit problematic to me, but at the same time, it also means that building Core using its own CMake file will not have this support. But not sure how much it matters?

I could also just put this script in the Kotlin repo, but it also seems awkward if all SDK's need to implement this. On the other side, having this script in the Core repo without it being documented or used also means it will just be forgotten. 

Given that there isn't a clear advantage to having this code in the Core repo, I'll probably just move it to Kotlin for now and then we can always move it here if it make sense at some point.

Reply:
@cmelchior Did you intend to close this PR?